diff --git a/node_modules/@tarojs/rn-runner/src/index.ts b/node_modules/@tarojs/rn-runner/src/index.ts
index b4cf8d5..0aabde6 100644
--- a/node_modules/@tarojs/rn-runner/src/index.ts
+++ b/node_modules/@tarojs/rn-runner/src/index.ts
@@ -61,13 +61,16 @@ export default async function build (_appPath: string, config: any): Promise<any
       cliParams.push('--port', config.port)
     }
     try {
+      const command = [npxCmd, 'react-native', 'start', '--custom-log-reporter-path', '@tarojs/rn-supporter/TerminalReporter'].concat(cliParams);
+      console.log('即将执行的命令:', command);
       spawn(npxCmd, [
         'react-native',
         'start',
         '--custom-log-reporter-path',
         '@tarojs/rn-supporter/TerminalReporter'
       ].concat(cliParams), {
-        stdio: 'inherit'
+        stdio: 'inherit',
+        shell: true
       })
       if (config.qr) {
         previewDev({
@@ -105,6 +108,8 @@ export default async function build (_appPath: string, config: any): Promise<any
     fse.ensureDirSync(assetsDest)
 
     try {
+      const command = [npxCmd, 'react-native', 'start', '--custom-log-reporter-path', '@tarojs/rn-supporter/TerminalReporter'].concat(cliParams);
+      console.log('即将执行的命令:', command);
       spawn(npxCmd, [
         'react-native',
         'bundle',
